Eastern-Research-Group/csb-rebate-forms-app
# Feature/update 2024 nces

## Related Issues:
* CSBAPP-503

## Main Changes:
* Update 2024 NCES file (A000001 school district added).
_NOTE: the order of the school districts have changed since the last update, but we've confirmed via a diff tool that the only change is the new school district being added._

## Steps To Test:
1. Navigate to the 2024 FRF
2. Enter "A000001" for the NCES lookup.
3. Ensure the school district info is displayed.
4. Everything else should function normally.
